target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -335,11 +335,19 @@ wndr3700v5)
 wt3020)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
 	;;
-y1|\
+y1)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	set_usb_led "$board:blue:usb"
+	set_wifi_led "$board:blue:wifi" "wlan1"
+	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wifi5g" "wlan0" "link tx rx"
+	ucidef_set_led_netdev "lan" "lan" "$board:blue:lan" "eth0.1" "tx rx"
+	ucidef_set_led_netdev "wan" "wan" "$board:blue:internet" "eth0.2" "tx rx"
+	;;
 y1s)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
```

Comment:
This shouldn't be required. The LED is already handled by target/linux/ramips/base-files/etc/diag.sh.